rust/arrow/benches/equal.rs

```diff
@@ -38,10 +38,10 @@ fn add_benchmark(c: &mut Criterion) {
     let arr_a_nulls = create_primitive_array::<Float32Type>(512, 0.5);
     c.bench_function("equal_nulls_512", |b| b.iter(|| bench_equal(&arr_a_nulls)));
 
-    let arr_a = create_string_array(512, 0.0);
+    let arr_a = create_string_array::<i32>(512, 0.0);
```

Comment:
Not part of the PR - but I notice this benchmark is strange / not very representative as it is testing two of the same arrays to be equal.

Reply:
Yeah, I see what you mean, we can create another array, and compare the 2

Reply:
Yeah, and also the seeds need to be different in that case, earlier I found a benchmark which created two same arrays, but as the code uses the same `42` seed, it still had to equivalent arrays :shrug: